phoenix-core/src/main/java/org/apache/hadoop/hbase/ipc/PhoenixRpcScheduler.java

```diff
@@ -40,28 +40,34 @@
     private static final int DEFAULT_MAX_CALLQUEUE_LENGTH_PER_HANDLER = 10;
 
     private RpcScheduler delegate;
-    private int minPriority;
-    private int maxPriority;
-    private RpcExecutor callExecutor;
+    private int indexPriority;
+    private int metadataPriority;
+    private RpcExecutor indexCallExecutor;
+    private RpcExecutor metadataCallExecutor;
     private int port;
 
-    public PhoenixIndexRpcScheduler(int indexHandlerCount, Configuration conf,
-            RpcScheduler delegate, int minPriority, int maxPriority) {
+    public PhoenixRpcScheduler(int indexHandlerCount, int metadataHandlerCount, Configuration conf,
```

Comment:
@JamesRTaylor 
>  I believe this was based on the range of priorities we were reserving. Both are just 1 now.

Why don't you agree with the future proofing of a range of priorities? I think I put forward a good case for them elsewhere.. maybe I didn't see your comments otherwise?